src/csvcubed/utils/sparql_handler/csvw_inspector.py

```diff
@@ -69,12 +97,11 @@ def get_primary_catalog_metadata(self) -> CatalogMetadataResult:
             f"Could not find catalog metadata in primary graph '{self.primary_graph_uri}'."
         )
 
-    def get_column_definitions_for_csv(self, csv_url: str) -> List[ColumnDefinition]:
+    def get_table_info_for_csv_url(self, csv_url: str) -> TableSchemaPropertiesResult:
         """
-        Returns the `ColumnDefinition`s for a given csv file, raises a KeyError if the csv_url
```

Comment:
Can you go back to the comment you've replaced here.

Reply:
Bit confused about this - I don't think we've changed this comment? It's the same as the one that's currently in csvw_state.py on main

Reply:
The previous comment was:

> Returns the `ColumnDefinition`s for a given csv file, raises a KeyError if the csv_url is not associated with a ColumnDefinition.

And this PR replaces it with:

> Retrieves the stored result of the table schema properties cached property.

Reply:
"Retrieves the stored result of the table schema properties cached property" is the comment for `get_table_info_for_csv_url` not `get_column_definitions_for_csv`

Reply:
Ah ok, my bad